frontend-react/config/jest/fileTransform.js

```diff
@@ -0,0 +1,39 @@
+const path = require("path");
+
+const camelcase = require("camelcase");
+
+// This is a custom Jest transformer turning file imports into filenames.
+// http://facebook.github.io/jest/docs/en/webpack.html
+
+module.exports = {
```

Comment:
Oh, interesting how they handle this, though kind of unnecessary for our purposes.  Probably can remove in the subsequent cleanup since we already have the static asset mock, right?